cmd/node/config/enableEpochs.toml

```diff
@@ -156,9 +156,6 @@
     # FrontRunningProtectionEnableEpoch represents the epoch when the first version of protection against front running is enabled
     FrontRunningProtectionEnableEpoch = 1
 
-    # DisableOldTrieStorageEpoch represents the epoch when the old trie storage implementation will be disabled
-    DisableOldTrieStorageEpoch = 2
```

Comment:
is it ok to remove the DisableEpoch?

Reply:
Yes, because this "event" already happened on mainnet, so everybody will start without the old storage.

Reply:
ok